### PR TITLE
fix a failure unit test when running on 10.15

### PR DESCRIPTION
**Pull request by kaisong1990**
## Proposed changes

The change is to remove items in memory and keep those items in local cache(storage)

Describe what this PR is trying to do.

This PR is to address a unit test failure on 10.15 and above where we should not be able to get local cached items. However, the logic of query items is a combination of getting items from both local and memory which fails the unit test. This PR will help to remove the items in memory only to prove we cannot get items from local cache

## Type of change

- [ ] Feature work
- [ ] Bug fix
- [ ] Documentation
- [ ] Engineering change
- [x] Test
- [ ] Logging/Telemetry

## Risk

- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [x] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

### CODE REVIEW

**Comment by kaisong1990** [comment]
@rohitnarula7176 @oldalton Thanks for approving, when I upgrade to use xcode 10.15 in Travis, I got test failures on iOS target due to methods cannot be overwritten anymore. I used the swizzling way and looks like the issue got resolved. Would like to borrow your eyes to make sure this won't break anything else

File: IdentityCore/tests/util/MSIDApplicationTestUtil.m [hunk 1]
```diff
@@ -60,7 +61,22 @@ @interface UIApplication (TestOverride)
 #pragma clang diagnostic ignored "-Wobjc-protocol-method-implementation"
 @implementation UIApplication (TestOverride)
 
-- (BOOL)openURL:(NSURL *)url
++ (void)load {
```
Comment (antrix1989) [inline IdentityCore/tests/util/MSIDApplicationTestUtil.m:64]: nit: new line

Comment (kaisong1990) [reply IdentityCore/tests/util/MSIDApplicationTestUtil.m:64]: Good catch. Still in Swift style

File: IdentityCore/tests/util/MSIDApplicationTestUtil.m [hunk 2]
```diff
@@ -70,7 +86,7 @@ - (BOOL)openURL:(NSURL *)url
     return s_onOpenUrl(url, nil);
 }
 
-- (BOOL)canOpenURL:(NSURL *)url
+- (BOOL)TO_canOpenURL:(NSURL *)url
```
Comment (antrix1989) [inline IdentityCore/tests/util/MSIDApplicationTestUtil.m:89]: Rename `TO_canOpenURL` to "msidCanOpenURL". According to our codestyle, we always start with small letter and don't use underscore.

Comment (kaisong1990) [reply IdentityCore/tests/util/MSIDApplicationTestUtil.m:89]: Will update

**Pull request merged**